examples/src/test/java/io/numaproj/numaflow/examples/server/ServerTest.java

```diff
@@ -14,260 +14,256 @@
 import io.numaproj.numaflow.sinker.ResponseList;
 import io.numaproj.numaflow.sinker.SinkerTestKit;
 import io.numaproj.numaflow.sourcetransformer.SourceTransformerTestKit;
+import java.time.Instant;
```

Comment:
Why is it reformatting? Editor config is not used?

Reply:
Thank you! I tested a bit on my local. It was reformatting because 1. my IDE wasn't configured to respect editor config, and 2. some of the files in the examples folder were not formatted following editor config.

I configured my IDE to respect and also copied the `.editorconfig` file to examples folder, just so if future developers create a project using examples folder as root, the editor config is still respected by their IDE.